### PR TITLE
Add post on team offsite week on mallorca with postcards

### PR DESCRIPTION
**Pull request by dataduke**

### CODE REVIEW

File: _posts/2016/2016-03-17-team-orange-mallorca.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Developer team enjoys an innovative and productive offsite week on Mallorca"
```
Comment (BirgitBader) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: I'm sorry for this, but the title is too long.
We'd like it short and simple.
Suggestion: Productive team offsite on Mallorca

File: _posts/2016/2016-03-17-team-orange-mallorca.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Now we are back and ready to briefly tell you the tale of the fearless developers' journey to a mysterious island. Enjoy the multifaceted report!
+
+## Travelling together
+
+With much curiosity we first took some time off and enjoyed the amazing view over the beach skyline from the terrace of our finca, [Son Pages](http://sonpages.com). On day two, we visited the [Cuevas del Drach](http://www.cuevasdeldrach.com) and the beautiful yacht harbour at [Porto Cristo](http://www.portocristo.org/) to draw fresh inspiration. On the next day, we undertook a road trip with the team van to Sa Calobra in the north. We were delighted by the majestic mountains and completely captivated by the bluish tonalities of the Mediterranean Sea. Alongside the curvy mountain road and nested into the picturesque bay were various [Geocaching](https://www.geocaching.com) possibilites that we thankworthy explored with tremendous pleasure.
```
Comment (BirgitBader) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: we undertook a road trip > we went on a road trip

Comment (BirgitBader) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: This whole paragraph reads very romantic. Can we skip some shiny adjectives here and there?
Suggestion: We were impressed by the majestic mountains and the bluish tonalities of the Mediterranean Sea.
When cruising along the bay area, we also found various [Geocaching](https://www.geocaching.com) possibilities to explore.

File: _posts/2016/2016-03-17-team-orange-mallorca.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+{% image blog/blog-mallorca-1.jpg %}{% endimage %}
+
+## Living as a team
+
+Sharing daily life activities such as having breakfast, shopping, cooking and playing board games together was definitely a delightful way to become closer. Every morning we even had a wonderful one hour wake-up **fitness workout** by a fun-loving and very disciplined personal trainer. This kick-starting routine often blended nicely into the warming **sunrise over the palm island** – what a sight!
```
Comment (BirgitBader) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: I would prefer plain formatting without using bold. It's too much.

File: _posts/2016/2016-03-17-team-orange-mallorca.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Developer team enjoys an innovative and productive offsite week on Mallorca"
+date: "2016-03-17 16:03:17"
+icon: suitcase
+categories: working-at-epages, events
```
Comment (BirgitBader) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: working-at-epages, events > remove the comma

File: _posts/2016/2016-03-17-team-orange-mallorca.md (hunk 3 above)
Comment (dataduke) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: removed all bolds

File: _posts/2016/2016-03-17-team-orange-mallorca.md (hunk 2 above)
Comment (dataduke) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: rewritten suggested

File: _posts/2016/2016-03-17-team-orange-mallorca.md (hunk 1 above)
Comment (dataduke) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: rewritten suggested

File: _posts/2016/2016-03-17-team-orange-mallorca.md (hunk 4 above)
Comment (dataduke) [inline _posts/2016/2016-03-17-team-orange-mallorca.md]: done

**Pull request merged**